packages/syft/src/syft/service/user/user_warning.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
Q: Should  we delete this file as it is empty?

Reply:
Oh right !! added by mistake 😄